spring-data-neo4j/src/test/java/org/springframework/data/neo4j/queries/DerivedQueryTests.java

```diff
@@ -26,7 +13,13 @@
 
 package org.springframework.data.neo4j.queries;
 
-import static org.junit.Assert.*;
```

Comment:
We always star static imports. Standard imports are starred for more than 10 imports of the same package.